extensions/cornerstone-3d/src/utils/colormap/applyPreset.js

```diff
@@ -0,0 +1,137 @@
+import vtkColorTransferFunction from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction';
+import vtkPiecewiseFunction from '@kitware/vtk.js/Common/DataModel/PiecewiseFunction';
+
+import presets from './presets';
+
+// get preset from preset name
+function getPreset(name) {
+  return presets.find(preset => preset.name === name);
+}
+
+export default function applyPreset(actor, presetName) {
```

Comment:
What if I want to apply my own function. Can we register presets? Maybe we should be able  to register presets from the `cornerstone-3d` extenison's utility module.

Reply:
Interesting idea, I agree, but not currently in the scope